data/input/registry-migration.js

```diff
@@ -38,9 +38,9 @@ Object.keys(environments).forEach(network => {
   environments[network] = {
     ...environments[network],
     governor:       governor[network],
-    court:          requireOutput(`court.${network}`, court => court.court.address),
-    disputeManager: requireOutput(`court.${network}`, court => court.disputeManager.address),
-    bondedToken:    requireOutput(`minime.${network}`, tokens => tokens.ANJ.address),
+    court:          requireOutput(`court.${network}`, getAddressIfDefined('court')),
```

Comment:
It's not a big deal, but without knowing the definition of `getAddressIfDefined()` or `requireOutput()`, seeing this raises quite a few eyebrows.

Two thoughts:

- Do we really need the `callback()` parameter in `requireOutput()`? Could we not do it synchronously?
- It might be clearer to "flatten" `getAddressIfDefined()`, so we use it like:

```
requireOutput(`court.${network}`, court => getAddressIfDefined(court, 'court'))
```

Reply:
> It might be clearer to "flatten" getAddressIfDefined(), so we use it like:

What’s the advantage of doing it like this?

Reply:
To me, it's more clear what's happening at a glance.

```js
requireOutput(`court.${network}`, getAddressIfDefined('court'))
```

vs. 

```js
requireOutput(`court.${network}`, court => getAddressIfDefined(court, 'court'))
```

In the latter, you can immediately tell the second parameter is a callback, and what object you're getting the address from (more context).

Reply:
Yea, it's kind of magically resolving what needs to be indexed